server/build.gradle

```diff
@@ -24,11 +24,19 @@ dependencies {
 distributions {
     main {
         contents {
-	    into('client') {
-		from { project(':client').build.outputs.files }
-		fileMode = 0755
-	    }
-	}
+            // Put the client artifacts (HTML, CSS, and JS generated
+            // by Angular) in the `lib/client/public` directory. This
```

Comment:
So the JS code that WebPack converts from TS (e.g., `User.component.ts`) runs in the browser (along with all the Angular TS/JS library code). Who does things like assemble the HTML fragments (e.g., `User.component.html`) along with all the necessary templating substitution? Is that also WebPack, or is that the Angular code doing that at runtime or…?